.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: 'We really appreciate your effort to provide feedback. Before opening a new
+  issue, please make sure that this case is not already reported in GitHub as an
+  issue or in StackOverflow as a question.'
+
+---
+
+**Environment**
+Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
+ - CLI: 
+ - Cross-platform modules:
+ - Android Runtime:
+ - iOS Runtime:
+ - Plugin(s):
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. Please, explain whether it's a build time error or a runtime error. More detailed logs can be easily obtained by following the instructions in this guide: https://docs.nativescript.org/get-support#how-to-obtain-diagnostic-reports. -->
+
+**To Reproduce**
+<!-- Add commands used or steps taken to reproduce the behaviour. -->
+
+**Expected behavior**
+
+**Sample project**
+<!-- If possible, provide a link from the [Playground](https://play.nativescript.org) with reproduction of the problem. If not, consider attaching a sample project or link to a repository with such project. -->
```

Comment:
Webpack currently doesn't work in the Playground.

Reply:
Good point